client/gamescreen.js

```diff
@@ -0,0 +1,106 @@
+(function() {
+    'use strict';
+
+    var Game = window.Game = window.Game || {};
+
+    //
+    // The main state holding class for the game.
```

Comment:
We can make a separate rendering class later if necessary, but this is the cleanest, simplest way to do it for now.